frontend/src/types/nlp-entity.types.ts

```diff
@@ -19,6 +19,7 @@ export interface INlpEntityAttributes {
   lookups: Lookup[];
   doc?: string;
   builtin?: boolean;
+  weight: number;
```

Comment:
```suggestion
  weight?: number;
```

Hi @MohamedAliBouhaouala , do think that weight can be optional if the DTO defined as optional from the API side ?
![image](https://github.com/user-attachments/assets/04746b1d-fd6d-43c8-9f21-636b681d6dbf)